src/rhsm/connection.py

```diff
@@ -738,6 +738,39 @@ def _load_ca_certificates(self, context: ssl.SSLContext) -> None:
         else:
             log.warning("Unable to load any CA certificate from: %s" % self.ca_dir)
 
+    def _get_tls_handshake_info(self) -> str:
+        """
+        Gets TLS handshake information for debugging and Post Quantum Cryptography testing.
+        Extracts information about the negotiated TLS connection: protocol version,
+        cipher suite, and if available (Python 3.15+), the key exchange group.
+        """
+        try:
+            handshake_info = []
+            # Get cipher suite information
+            # cipher() returns a 3-tuple: (cipher_name, tls_version, secret_bits)
+            CIPHER_NAME_POS = 0
+            TLS_VERSION_POS = 1
```

Comment:
The variables in the function should be lower case ... if following comment will be accepted, then we could cutoff these two variables.